src/Listeners/KernelEventsSubscriber.php

```diff
@@ -157,7 +155,7 @@ public function onKernelPostControllerArguments(ControllerArgumentsEvent $event)
      */
     public function onKernelRequest(RequestEvent $event): void
     {
-        if (!$event->isMasterRequest()) {
+        if (!$event->isMainRequest()) {
```

Comment:
If the isMainRequest isn't available in Symfony 5 it will break the package instead of extend the compatibility. In this case a new major version is needed. Maybe a test case can make prove it.